docs/Project.toml

```diff
@@ -1,5 +1,6 @@
 [deps]
+DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
```

Comment:
I'd say we usually avoid putting the package here, since it might happen that you silently end up having the latest release, not the `master` version of the package in the docs environment:

```suggestion
```

Reply:
Oooh, sorry for repeating the same mistake (https://github.com/JuliaDocs/DocStringExtensions.jl/pull/118#discussion_r675958575). I remember that comment, but I have mistakenly committed that line. 🙇